src/modules/formBuilder/components/FormBuilder.tsx

```diff
@@ -107,11 +108,96 @@ const FormBuilder: React.FC<FormBuilderProps> = ({
           proceeding?
         </Typography>
       </ConfirmationDialog>
-      <Box sx={{ display: 'flex' }}>
+      {selectedItem && (
```

Comment:
GH is making this change look bigger than it is -- it's just a rearrangement of these components due to moving the Form Palette over to the right-hand side.